kof-operator/webapp/collector/src/components/pages/collectorPage/components/collector-list/CollectorsTable.tsx

```diff
@@ -52,8 +53,19 @@ const CollectorsTable = ({ cluster }: { cluster: Cluster }): JSX.Element => {
             </TableRow>
           </TableHeader>
           <TableBody>
-            {cluster.pods.map((pod) => (
-              <CollectorRow key={pod.name} pod={pod} />
+            {cluster.customResource.map((cr) => (
+              <>
```

Comment:
Did you check console for any react complains about unique key for repeated elements?